packages/react-composites/src/composites/ChatComposite/file-sharing/FileCardGroup.tsx

```diff
@@ -19,11 +19,13 @@ const fileCardGroupClassName = mergeStyles({
   flexFlow: 'row wrap',
   '& > *': {
     margin: '0.5rem',
-    marginLeft: '0 rem',
     marginRight: 'auto'
   },
-  ':not(:first-child)': {
-    marginLeft: '0.5rem'
+  /**
+   * margin for children is overriden by parent stack, so adding left margin for each child
```

Comment:
Thanks for adding this comment, it details well what is happening but not why -- why does each child (but not the first) require a margin left?